Tella/Data/Database/TellaData.swift

```diff
@@ -119,18 +129,23 @@ class TellaData : ObservableObject {
     }
     
     func deleteGDriveServer(serverId: Int) {
-        GDriveRepository().signOut()
         database.deleteGDriveServer(serverId: serverId)
         reloadServers()
     }
     
-    func deleteNextcloudServer(serverId: Int) -> Bool {
+    @discardableResult
+    func deleteNextcloudServer(serverId: Int) -> Result<Void,Error> {
         // signOut
```

Comment:
Do we really need the `Result<Void,Error> `here ? I see that we are not using the same pattern for the deletion in the other connection ?